appeals/web/src/server/appeals/appeal-details/final-comments/redact/redact.mapper.js

```diff
@@ -0,0 +1,109 @@
+import { appealShortReference } from '#lib/appeals-formatter.js';
+import { preRenderPageComponents } from '#lib/nunjucks-template-builders/page-component-rendering.js';
+import { wrapComponents, simpleHtmlComponent } from '#lib/mappers/index.js';
+import { redactInput } from '../../interested-party-comments/redact/components/redact-input.js'; // TODO: move to shared location
```

Comment:
`../../representations/common` might be a good place?

Reply:
Done :)